data/json/construction.json

```diff
@@ -1245,6 +1246,7 @@
             [ [ "2x4", 3 ] ],
             [ [ "rope_6", 2 ], [ "vine_30", 1 ] ]
         ],
+        "pre_note" : "Must be between palisade walls to function, at least one wall must have an adjacent rope & pulley system.",
```

Comment:
Note itself looks good – grammatically, you should either change the period to a comma, or add an "add" after it.